ui/src/app/shared/services/environments.service.ts

```diff
@@ -114,13 +115,19 @@ export class EnvironmentsService {
         return this.http.get('/api/environments/' + environmentId + '/changelog?limit=100').pipe(map(d => d as ChangelogEntry[]));
     }
 
+    public getRegisteredApplications(environmentId: string): Promise<ApplicationInfo[]> {
```

Comment:
I think this method should better go into applications.service.ts. And it has an ugly empty line at its end :-)